pkg/cluster/samples.go

```diff
@@ -61,14 +61,23 @@ func (m *manager) disableOperatorHubSources(ctx context.Context) error {
 				Name:     "certified-operators",
 				Disabled: true,
 			},
+			{
```

Comment:
Does not look related to this PR

Reply:
Correct; the first few commits in this PR are like that.